go/analyzer/staticcheck/util/util.go

```diff
@@ -0,0 +1,47 @@
+package util
+
+import (
+	"fmt"
+	"sort"
+
+	"golang.org/x/tools/go/analysis"
+	"honnef.co/go/tools/analysis/lint"
+	"honnef.co/go/tools/quickfix"
+	"honnef.co/go/tools/simple"
+	"honnef.co/go/tools/staticcheck"
+	"honnef.co/go/tools/stylecheck"
+)
+
+var StaticcheckAnalyzers = [][]*lint.Analyzer{
+	quickfix.Analyzers,
+	simple.Analyzers,
+	staticcheck.Analyzers,
+	stylecheck.Analyzers,
+}
+
+var analyzerMap, AnalyzerSlice = func() (map[string]*analysis.Analyzer, []*analysis.Analyzer) {
```

Comment:
We may not need `AnalyzerSlice`, as the only place that uses it is the `all_checks_printer`, which can also iterate through the `analyzerMap`.

Reply:
The only problem with the map is that its order is not deterministic. As such, the output (could) be different every time.